### PR TITLE
Adding known issue for search query error

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/22571

Zack, thank you so much for the detailed issue and thanks for your patience! When you have an opportunity, will you review this PR?

### CODE REVIEW

**Review by zlayne — APPROVED** [review]

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -121,6 +121,18 @@ When you update the storage size in the `searchcustomization` CR, the PVC config
 oc edit pvc <storageclassname>-search-redisgraph-0
 ----
 
+[#search-query-issue]
+=== Search query parsing error
+//2.5:22391 
+
+If an environment becomes large and require more test for scaling, the search queries can timeout which results in an parsing error message being displayed. This error is displayed after 30 seconds of waiting for a search query.
```
Comment (oafischer) [inline release_notes/known_issues.adoc]: Check 'require' and 'test' again, I assume both are meant to be plural.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -121,6 +121,18 @@ When you update the storage size in the `searchcustomization` CR, the PVC config
...
+
+Extend the timeout time with the following command:
+
+---
+kubectl annotate route multicloud-console haproxy.router.openshift.io/timeout=Xs
```
Comment (oafischer) [inline release_notes/known_issues.adoc:133]: Add missing '$'

**Review by oafischer — COMMENTED** [review]
Left some comments for you to go over.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc]: You're right, great catch thanks!

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc:133]: Thanks Oliver! The `$` isn't required for the command. In the issue the developer provided it without, so keeping it consistent with what was provided. I usually don't add the `$`. 

This is a good topic to bring up during our doc arch call on Thursday tho! It'll be good to receive a team consensus and be consistent across the docs.

Comment (oafischer) [reply release_notes/known_issues.adoc:133]: Didn't know that, thanks for explaining! In that case definitely leave it out, consistency is always good.

**Review by oafischer — APPROVED** [review]
Looks good!

**Pull request merged**